docs/source/api/v2/asns.rst

```diff
@@ -110,12 +110,12 @@ Creates a new :abbr:`ASN (Autonomous System Number)`.
 
 Request Structure
 -----------------
-:asn:        The value of the new :abbr:`ASN (Autonomous System Number)`
-:cachegroup: An optional field which, if present, is a string that specifies the :ref:`cache-group-name` of a :term:`Cache Group` to which this :abbr:`ASN (Autonomous System Number)` will be assigned
+:asn:		The value of the new :abbr:`ASN (Autonomous System Number)`
+:cachegroup:	An optional field which, if present, is a string that specifies the :ref:`cache-group-name` of a :term:`Cache Group` to which this :abbr:`ASN (Autonomous System Number)` will be assigned
```

Comment:
Same question here

Reply:
Aligned fields with spaces in c2b73f08e9.